benchmarks/degredation_models.py

```diff
@@ -0,0 +1,549 @@
+import pybamm
+import numpy as np
+
+
+def compute_discretisation(model, param):
+    var_pts = {
+        pybamm.standard_spatial_vars.x_n: 20,
+        pybamm.standard_spatial_vars.x_s: 20,
+        pybamm.standard_spatial_vars.x_p: 20,
+        pybamm.standard_spatial_vars.r_n: 30,
```

Comment:
Are you using 30 in the particles for some reason (e.g. convergence)? Otherwise might be better to keep the default (20).